src/components/Main.jsx

```diff
@@ -1,19 +1,40 @@
-
+import Twitter from "../image/twittericon.png";
```

Comment:
As a naming convention I would recommend something like:
```suggestion
import Twitter from "../images/icons/twitter.png";
```